.goreleaser.yaml

```diff
@@ -2,49 +2,69 @@ before:
   hooks:
     - make release-notes
 builds:
-- id: qbec
-  binary: qbec
-  main: ./
-  ldflags:
-  - -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.goVersion={{.Env.GO_VERSION}}
-  env:
-  - CGO_ENABLED=0
-  goos:
-  - linux
-  - darwin
-  - windows
-  goarch:
-  - amd64
-- id: jsonnet-qbec
-  binary: jsonnet-qbec
-  main: cmd/jsonnet-qbec/main.go
-  ldflags:
-  - -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.goVersion={{.Env.GO_VERSION}}
-  env:
-  - CGO_ENABLED=0
-  goos:
-  - linux
-  - darwin
-  - windows
-  goarch:
-  - amd64
+  - id: qbec
+    binary: qbec
+    main: ./
+    ldflags:
+      - -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.goVersion={{.Env.GO_VERSION}}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+  - id: jsonnet-qbec
+    binary: jsonnet-qbec
+    main: cmd/jsonnet-qbec/main.go
+    ldflags:
+      - -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.goVersion={{.Env.GO_VERSION}}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
 archives:
   - name_template: "{{.ProjectName}}-{{.Os}}-{{.Arch}}"
     format: tar.gz
     format_overrides:
-    - goos: windows
-      format: zip
+      - goos: windows
+        format: zip
+
+brews:
+  - name: qbec
+    github:
+      owner: splunk
+      name: homebrew-tap
+    url_template: https://github.com/splunk/qbec/releases/download/{{.Tag}}/{{.ArtifactName}}
+
+    commit_author:
```

Comment:
I don't think this is needed based on the goreleaser example. I believe it'll take commit ownership from the person who's change triggered the update.

Reply:
Actually, it'll take it from the API key (Krishnan) who merges all the changes anyways.

Reply:
The default is not the author but the goreleaserbot. See https://github.com/harsimranmaan/homebrew-tap/commit/b397e234e4710b78716e69f55514769f9cc046f2 where I did the test. I'd prefer it to be a splunk user.

Reply:
If it's a Splunk user, make it Krishnan

Reply:
The committer is the author not the publisher. I am not sure there’s any value in changing this unless it is a bot.

Reply:
For consistency with the rest of the project, can you make it Krishnan since he's the project owner.

Reply:
@harsimranmaan If you can change this authorship to Krishnan's identity, then I'll sign off!

Reply:
I am not sure that there's a compelling reason to.

Reply:
Like I said, he's the only person allowed to directly commit/push to this project, so from a consistency perspective, to me it makes sense that listing the authorship here would match the project owner. All commits in this project are under his identity.

Reply:
The example that seems weird to me is if I made an MR, Krishnan merges it, and then the tap update author shows up as you. That's what we would have here. At least if it were Krishnan, it's consistent with the push action.

Reply:
or go back to using the bot so we know that it's an automated action since we can't get the true contributor information